ecf/hurricane/stats/jevs_hurricane_global_ens_tropcyc_stats.ecf

```diff
@@ -32,6 +32,7 @@ module load grib_util/$grib_util_ver
 module load prod_util/${produtil_ver}
 module load prod_envir/${prodenvir_ver}
 module load imagemagick/${imagemagick_ver}
+module load libfabric/${libfabric_ver}
```

Comment:
Please test the following recommendation:
1- replace `module purge` with `module reset`
2- remove line 35
By using module reset instead of module purge you will no longer need to load the libfabric module.